activerecord/test/cases/relations_test.rb

```diff
@@ -220,7 +220,7 @@ def test_finding_with_assoc_order
   end
 
   def test_finding_with_arel_assoc_order
-    topics = Topic.order(Arel.sql("id") => :desc)
+    topics = Topic.order("id" => :desc)
```

Comment:
This and the following are specifically testing `Arel.sql`.. overshoot on a find & replace?